.bldr.toml

```diff
@@ -957,6 +960,12 @@ paths = [
   "lib/user/*"
 ]
 
+[automate-cs-ocid]
```

Comment:
This placement was wrong, the pipeline got failed. https://buildkite.com/chef-oss/chef-automate-main-verify/builds/29337#018944d2-1580-4c7f-8b72-2c3e9c62a5bb

Reply:
Done. Changes can be checked here https://github.com/chef/automate/pull/8059